macos/package_files/3.9.3/preinstall.sh

```diff
@@ -17,6 +17,27 @@ else
     launchctl setenv WAZUH_PKG_UPGRADE true
 fi
 
+if [ WAZUH_PKG_UPGRADE ]; then
+    mkdir /Library/Ossec/config_files/
+    cp -r /Library/Ossec/etc/{ossec.conf,client.keys,local_internal_options.conf,shared} /Library/Ossec/config_files/
+fi
+
+if [ WAZUH_PKG_UPGRADE ]; then
+    VERSION=`pkgutil --info com.wazuh.pkg.wazuh-agent | grep version |  cut -d" " -f2- | rev|cut -d"-" -f2- | rev`
```

Comment:
I think this block can be simplified. You can check if the package `com.wazuh.pkg.wazuh-agent-etc` is installed or not, and if it is installed, just remove it. You don't need to check the version if this package isn't necessary from Wazuh v3.9.3.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/57e1e49aa5d206859120786b6ec4357d6b9bb8ad